pkg/tss/constants.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package tss provides TPM 1.2/2.0 core functionality and
+// abstraction layer for high-level functions
+package tss
+
+// Locality type
+type Locality byte
```

Comment:
group types with their constants

Reply:
Removed Locality. If needed - fall back to go-tpm library.
Rest done.